examples/cmake/FindLibtorrentRasterbar.cmake

```diff
@@ -118,12 +118,14 @@ function(_try_generic_mode)
 		return()
 	endif()
 
-	if(NOT Boost_SYSTEM_FOUND)
-		find_package(Boost QUIET REQUIRED COMPONENTS system)
+	find_package(Boost QUIET REQUIRED)
+	if (Boost_MAJOR_VERSION LESS_EQUAL 1 AND Boost_MINOR_VERSION LESS 69)
+		if (NOT Boost_SYSTEM_FOUND)
+			find_package(Boost QUIET REQUIRED COMPONENTS system)
+		endif()
+		set(LibtorrentRasterbar_LIBRARIES Boost::system ${CMAKE_THREAD_LIBS_INIT})
```

Comment:
it looks suspicious to move this whole `set()` call to within the if-block. with boost >= 1.69, do we not need to include `${CMAKE_THREAD_LIBS_INIT}`?

Reply:
you're right, now I've separated `${CMAKE_THREAD_LIBS_INIT}` out.